### ISSUE TITLE
Team Video Statistics shouldn't include subtitles/languages without a version

### ISSUE THREAD

**Issue by darren-bridenbeck** [pculture/unisubs#2346]
Currently our Team Video Statistics includes "invisible" languages from the UI, that do not have any saved subtitle versions.

The expected behavior is that statistics are only generated for subtitles / languages that are visible within a team, and have a saved version.

**Comment by bendk** [comment]
I think one reason that we create a subtilte language object is because a task was created for that language.  If there's a task, but no version saved, should that be included?

**Comment by PCF-Testing** [comment]
No, tasks without any subtitles associated with them should not count towards the subtitles stats.

The major source of empty subtitles is users starting to subtitle in a language (i.e. opening the editor) and closing it without saving anything. This creates a language that has 0 versions;these should not count.
